migration_toolkit_for_containers/index.adoc

```diff
@@ -0,0 +1,62 @@
+[id='Migration-toolkit-for-containers-overview']
+= Migration toolkit for containers overview
+include::modules/common-attributes.adoc[]
+:context: Migration-toolkit-for-containers-overview
+
+toc::[]
+
+You can migrate stateful application workloads between {product-title} 4 clusters at the granularity of a namespace by using the MigrationToolkit for Containers (MTC). To know more about MTC see the xref:../migration_toolkit_for_containers/about-mtc.adoc#about-mtc[Understanding MTC].
```

Comment:
* MigrationToolkit -> Migration Toolkit
* To know more -> To learn more
* see the -> see

Reply:
acked